### DISCUSSION TITLE
Applications using Elements

### DISCUSSION THREAD

**Discussion by redtide** [cycfi/elements#352] [Show and tell]
Following a list of known applications using Elements; if you know others please let us know.

| Name | Description | License
| --- | --- | ---
| [elements-custom-controls] | Various custom controls | FOSS
| [sfizz] | sfz sampler (experimental version using Elements) | FOSS
| [ascend-vpu] | Ascend VPU | Free

[elements-custom-controls]: https://github.com/elements-custom-controls/controls
[sfizz]: https://github.com/jpcima/sfizz/tree/editor%2Belements%2Bui/editor
[ascend-vpu]: https://www.cycfi.com/2021/07/ascend-vpu-v1-1-2/

**Comment by na2axl** [comment]
[Amplitude](https://amplitudeaudiosdk.com)

<img width="1389" alt="image" src="https://github.com/cycfi/elements/assets/14062034/56e362f6-afb0-431d-aa00-01d07b92f560">

**Description:**
Amplitude Audio SDK is a free and open-source audio engine optimized for games. The official Amplitude projects authoring tool a.k.a **Amplitude Studio** is a multi-platform software built using Elements.

**License:**
_Amplitude Audio SDK_ - Free under Apache-2.0
_Amplitude Studio_ - Commercial

**Reply by djowel** [reply]
Wonderful!

**Comment by djowel** [comment]
Here's Ascend VPU:

![image](https://github.com/cycfi/elements/assets/568094/02b2282b-87c6-4812-9f25-d54fdaebfd2f)

Ascend VPU is an audio plugin for modeling the sonic characteristics of electric guitar pickups. Designed for multichannel pickups with wide and flat frequency response such as the [Nu Multichannel pickup](https://www.cycfi.com/projects/nu-series-v2/), or the [Roland-GK](https://www.roland.com/V-Guitar/about.html) divided pickup, the plugin makes it possible to reproduce the frequency response of various pickup positions (bridge, middle, neck) without actually moving the pickup. The technical details are elaborated in the five part Virtual Pickups series: [Part1](https://www.cycfi.com/2014/07/virtual-pickup-placement-part-1/), [Part2](https://www.cycfi.com/2014/07/virtual-pickup-placement-part-2/), [Part3](https://www.cycfi.com/2014/08/virtual-pickups-part-3/), [Part4](https://www.cycfi.com/2020/11/virtual-pickups-part-4/) and [Part5](https://www.cycfi.com/2020/11/virtual-pickups-part-5/).

**Comment by klytje** [comment]
[AUSAXS](https://github.com/AUSAXS/AUSAXS) is a small-angle X-ray scattering analysis software developed from scratch by me & my group at Aarhus University. The basic idea is to validate high-resolution atomic structures by analytically calculating their theoretical scattering profile and comparing it to a measurement (bottom left figure). The scope of the project was expanded to also support comparisons of SAXS measurements against cryogenic electron microscopy 3D charge-distribution maps (right figures). 

![image](https://github.com/cycfi/elements/assets/16706862/ed54c237-7c6c-456a-84e2-0381fdc80b9a)
![image](https://github.com/cycfi/elements/assets/16706862/c26a1348-86f4-4f4f-b31c-cecce7f35cf2)

I'm not super happy with the current GUIs, but they are functional and that's good enough for now. Hopefully I'll get around to updating them before I finish my project.
